src/app/submission/sections/upload/file/license-type/section-upload-file-license-type.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="d-flex flex-row" style="max-width: fit-content;">
+    <a [href]="licenseTypeURI" target="_blank">
+        <img [src]="imageToDisplay" [style]="'max-height:' + maxHeight + 'px;'">
```

Comment:
Maybe overkill for this feature but could be of interest to keep in mind : https://angular.io/guide/image-directive